lib/users.ts

```diff
@@ -36,13 +36,126 @@ export interface CreateResponse extends BaseResponse {
   status: string;
 }
 
-export interface GetResponse extends BaseResponse {
+interface User {
   user_id: UserID;
   name: Name;
   emails: Email[];
   phone_numbers: PhoneNumber[];
   webauthn_registrations: WebAuthnRegistration[];
+  providers: OAuthProvider[];
   status: string;
+  created_at: Date;
```

Comment:
```suggestion
  user_id: UserID;
  created_at: Date;
  status: string;
  name: Name;
  emails: Email[];
  phone_numbers: PhoneNumber[];
  providers: OAuthProvider[];
  webauthn_registrations: WebAuthnRegistration[];
```

I think this order makes more "sense" if we aren't going to go with alphabetical, it is roughly status>information>auth methods.

Reply:
done - https://github.com/stytchauth/stytch-node/pull/97/commits/4f4f647e27f216b63d2dc6e94aac1dd10d2f1224